xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2953,7 +2963,8 @@ void CVideoPlayer::HandleMessages()
             m_ChannelEntryTimeOut.SetInfinite();
             CloseDemuxer();
             m_playSpeed = DVD_PLAYSPEED_NORMAL;
-            m_newPlaySpeed = DVD_PLAYSPEED_NORMAL;
+            m_processInfo->SetSpeed(1.0);
+            m_processInfo->SetTempo(1.0);
```

Comment:
That is the pvr channel switch code that will go away completely (hopefully soon)